evm/src/main/java/org/hyperledger/besu/evm/gascalculator/CancunGasCalculator.java

```diff
@@ -22,13 +24,27 @@
  *   <LI>Data gas for EIP-4844
  * </UL>
  */
-public class CancunGasCalculator extends LondonGasCalculator {
+public class CancunGasCalculator extends ShanghaiGasCalculator {
+
+  /** Instantiates a new Cancun Gas Calculator. */
+  public CancunGasCalculator() {
+    this(KZG_POINT_EVAL.toArrayUnsafe()[19]);
+  }
+
+  /**
+   * Instantiates a new Cancun Gas Calculator
+   *
+   * @param maxPrecompile the max precompile
+   */
+  private CancunGasCalculator(final int maxPrecompile) {
+    super(maxPrecompile);
+  }
 
   private static final long TLOAD_GAS = WARM_STORAGE_READ_COST;
   private static final long TSTORE_GAS = WARM_STORAGE_READ_COST;
 
   private static final long DATA_GAS_PER_BLOB = 1 << 17;
-  private static final long TARGET_DATA_GAS_PER_BLOCK = 1 << 18;
+  private static final long TARGET_DATA_GAS_PER_BLOCK = 393216L;
```

Comment:
Since it's not a simple 1-shift, a hex representation would make it clearer it's a round hex number.  `0x60000`